### DISCUSSION TITLE
Mark Zuckerberg的裁員信帶給我的反思

### DISCUSSION THREAD

**Discussion by xxrjun** [1101-GS4719-C/Python-Community-GS4719-C#1171] [General]
> 去年 11 月 Meta 裁了約 11,000 人，占公司總數的 13%。
>
> 上週(3/14)宣布第二輪裁員，影響也差不多 1 萬人，並且關閉了近 5000 個職缺。
> 在思考大環境的影響之前，[Zuckerberg 給 Meta 員工的信](https://about.fb.com/news/2023/03/mark-zuckerberg-meta-year-of-efficiency/)讓我有了一些反思。

## 員工減少，有助於提高公司效率?

在這篇文中，雖然不乏有一些官腔的說法，但 Zuckerberg 也直接說明了 **"員工減少，有助於提高公司效率"**

本想說這是在傷口上撒鹽嗎，但是繼續往下看，我很快就被說服了。

這篇文有討論好幾個重點，不過其中對我影響比較大的是 **扁平化更快(Flatter is faster)** 以及 **越瘦越好(Leaner is better)** 的內容。

**扁平化更快(Flatter is faster)**

> "我們將通過取消多層管理使我們的組織更加扁平化，作為其中的一部分我們將要求許多經理成為個人貢獻者。
>
> 我們仍然認為管理每個人非常重要，所以一般來說我們不希望經理擁有超過 10 名直接下屬。"

我非常認同這段言論，管理層固然重要，但是管理層的數量過多或是單一團隊規模過大，反而會造成管理層的效率降低，也會造成管理層的溝通成本增加。我猜想未來科技公司的純管理階層會越來越少甚至趨近於零，並且全面改由有技術背景的人來管理。

**越瘦越好(Leaner is better)**

> "人們很容易認為，只要一個項目產生的價值超過其直接成本，它的淨效益就是正向的。"

第一句話就直接把我搞矇了，難道不是嗎? 但是 Zuckerberg 接下來這樣解釋

> "但那個項目需要一個領導者，所以也許我們可以從另一個團隊中挑選一位優秀的人才，
> 或者將一位優秀的工程師調入管理角色，這兩者會使人才被分散，並且創造出更多的管理層級。
>
> 這個項目團隊需要空間，也許會導致整個產品團隊被分散到多個樓層或時區，這將使得所有人的溝通變得更加困難。"

這段說明了領導者的角色會使得人才分散並降低溝通效率。

> "這個項目團隊需要筆記型電腦和人力資源福利，也許還需要招聘更多工程師，因此這將促使我們聘請更多的 IT、HR 和招聘人員，這樣這些組織將變得更不高效，對更高優先項目的反應也會變得更慢。
>
> 也許這個項目與另一個團隊的工作有所重疊，或者它可能在本應使用我們已經構建的通用基礎架構時卻構建了一個定制的技術系統，因此現在需要領導重點關注重複工作。
>
> 間接成本是累積的，而且人們很容易低估它們的影響。"

這段我其實有一點沒辦法深刻體會，不過大概就是想表達組織與團隊都應該越來越精簡、減少管理層級，然後持續的裁員，讓公司變得越來越瘦。還有間接成本長期而言也可能具有深遠的影響。

## Summary

總結來說我常常思考身為資訊管理學系的我，應該如何建構自己在資訊領域與管理領域的能力，不過看完這篇文後，加深了我想往資訊領域走的想法。因為未來直接進入管理層的可能大概會越來越小，要從技術能力開始，再往管理層走。不過科技產業也是越來越艱辛，所以勢必需要更多的努力。

**Comment by Hahahahbla** [comment]
Mark Zuckerberg 的言論確實引起了很多討論，特別是關於公司效率和組織結構的部分。Zuckerberg 的觀點反映了現代科技公司對於組織結構的再思考。精簡和扁平化不僅能提高效率，還能增強員工的自主性和責任感。在技術驅動的時代，技術背景的管理者能夠更好地引導和激發團隊的潛力。雖然裁員和精簡組織聽起來有些殘酷，但如果能夠通過這些措施實現更高的效率和更靈活的組織，最終可能會對公司的長期發展有利。

**Comment by NcuJerryKao** [comment]
你好，這篇文章的觀點值得深入探討。以下是我的建設性回答：

扁平化更快（Flatter is faster）
我完全同意這一點。過多的管理層級會增加決策時間和溝通成本，並且可能會造成信息傳遞的失真。取消多層管理，使經理成為個人貢獻者，能夠縮短決策鏈，提升靈活性和反應速度【1】。此外，這也讓有技術背景的人直接參與管理，更能理解技術需求和挑戰，從而做出更明智的決策【2】。

越瘦越好（Leaner is better）
Zuckerberg 提出的"間接成本"概念是重點。每個項目不僅需要直接成本，還有隱性的間接成本，如人力資源、設備和管理層級的增加。這些成本在長期內會累積，降低整體效率【3】。此外，將優秀的技術人才轉為管理角色，可能會造成核心技術工作的減少和分散，影響創新能力和技術發展【4】。因此，精簡組織結構，減少不必要的管理層級，可以提升整體效能。

總結來說，裁員和精簡組織的確可能帶來效率的提升，但需要注意保持核心技術團隊的穩定和技術領導力的發揮。這樣才能在保持高效運營的同時，持續創新和發展。

**Comment by titan92601** [comment]
我認為精簡管理和扁平化組織是十分重要的。作者認同減少管理層能提高效率，並警示領導者可能分散人才並降低溝通效率。同時，他提及間接成本的累積影響，建議持續精簡公司結構。作者從中獲得了對未來職涯方向的啟示，認為在科技領域建立技術能力後再考慮管理層職位。

**Comment by jylin34** [comment]
我認為Zuckerberg的觀點確實有其道理，就像馬斯克收購Twitter後進行大規模裁員，將公司組織扁平化並精簡人事一樣。然而，我認為Meta這次裁員的主要原因，還是由於過去兩年在低利率環境下過度招募人員所致。總而言之，在科技業或軟體工程領域，人數過多往往並非好事，反而會導致組織效率低下。

**Comment by LinChloe** [comment]
裁員不只能減少人力成本，也能使內部成員增加緊張感。就像老師行業出現了許多要教不教的害蟲，霸佔著位置不做事，整天怪學生太吵，無法上課。但如果這是來波裁員潮，那麼就不會有這種情況了。(沒錯，我真的想要某些老師被裁員==)
但裁員過度會使內部過度恐慌，大家無法專心做事，或是直接自動另找出路，新的人也不想往此公司發展。

**Comment by Shang621** [comment]
Mark Zuckerberg的裁員信讓我反思領導者應該如何正視困難決策。他在信中展示了坦誠和同情心，並承擔了公司的決策責任。這提醒我們，在艱難時刻，真誠和透明是建立信任和共鳴的關鍵。同時，這也讓我想到了團隊合作的重要性，如何支持和幫助那些受到影響的人，以及如何有效地應對變化。這封信給了我一個啟示，即在面對挑戰時，領導者應該勇於面對現實，同時保持人性化和同理心。

**Comment by sandyho0307** [comment]
Zuckerberg的裁員信反映了Meta在面對挑戰時所做出的戰略選擇。我認為雖然裁員對員工影響巨大，但如果能妥善處理，並在未來實現效率提升和創新發展，這對公司和員工都將是有利的。
